anyway/vehicle_type.py

```diff
@@ -0,0 +1,192 @@
+                VT.TRUCK_UPTO_4,
+                VT.PICKUP_UPTO_4,
+                VT.TRUCK_4_TO_10,
+                VT.TRUCK_12_TO_16,
+                VT.TRUCK_16_TO_34,
+                VT.TRUCK_ABOVE_34,
+                VT.BUS,
+                VT.TAXI,
+                VT.WORK,
+                VT.TRACTOR,
+                VT.MINIBUS,
+                VT.TRUCK_3_5_TO_10,
+                VT.TRUCK_10_TO_12,
+            ],
+            VehicleCategory.PRIVATE_DRIVER: [
+                VT.CAR,
+                VT.MOTORCYCLE_UPTO_50,
+                VT.MOTORCYCLE_50_TO_250,
+                VT.MOTORCYCLE_250_TO_500,
+                VT.MOTORCYCLE_ABOVE_500,
+            ],
+            VehicleCategory.LIGHT_ELECTRIC: [
+                VT.ELECTRIC_SCOOTER,
+                VT.MOBILITY_SCOOTER,
+                VT.ELECTRIC_BIKE
+            ],
+            VehicleCategory.CAR: [VT.CAR, VT.TAXI],
+            VehicleCategory.LARGE: [
+                VT.TRUCK_UPTO_4,
+                VT.PICKUP_UPTO_4,
+                VT.TRUCK_4_TO_10,
+                VT.TRUCK_12_TO_16,
+                VT.TRUCK_16_TO_34,
+                VT.TRUCK_ABOVE_34,
+                VT.BUS,
+                VT.WORK,
+                VT.TRACTOR,
+                VT.MINIBUS,
+                VT.TRUCK_3_5_TO_10,
+                VT.TRUCK_10_TO_12,
+            ],
+            VehicleCategory.MOTORCYCLE: [
+                VT.MOTORCYCLE_UPTO_50,
+                VT.MOTORCYCLE_50_TO_250,
+                VT.MOTORCYCLE_250_TO_500,
+                VT.MOTORCYCLE_ABOVE_500,
+            ],
+            VehicleCategory.BICYCLE_AND_SMALL_MOTOR: [VT.BIKE, VT.ELECTRIC_SCOOTER, VT.ELECTRIC_BIKE],
+            VehicleCategory.OTHER: [VT.BIKE, VT.TRAIN, VT.OTHER_AND_UNKNOWN]
+        }
+        return list(map(lambda x: x.value, category_vehicle_types[self]))
+
+    def contains(self, vt_code: int) -> bool:
+        # noinspection PyTypeChecker
+        if not isinstance(int, vt_code):
+            logging.warning(f"VehicleCategory.contains: {vt_code}:{type(vt_code)}: not int")
+            return False
+        return vt_code in self.get_codes()
+
+    def get_english_display_name(self):
+        english_vehicle_type_display_names = {
+            VC.PROFESSIONAL_DRIVER: _("professional driver"),
+            VC.PRIVATE_DRIVER: _("private driver"),
+            VC.LIGHT_ELECTRIC: _("light electric vehicles"),
+            VC.CAR: _("private car"),
+            VC.LARGE: _("large vehicle"),
+            VC.MOTORCYCLE: _("motorcycle"),
+            VC.BICYCLE_AND_SMALL_MOTOR: _("bicycle and small motor vehicles"),
+            VC.OTHER: _("other"),
+        }
+        try:
+            return english_vehicle_type_display_names[self]
+        except (KeyError, TypeError):
+            logging.exception(f"VehicleType.get_display_name: {self}: no display string defined")
+            return "no display name defined"
+
+
+VC = VehicleCategory
```

Comment:
why?

Reply:
Just a shorthand, but not really justified. Removed.